### ISSUE TITLE
CoinOR config.guess not able to guess for newer OSes

### ISSUE THREAD

**Issue by jpfeuffer** [OpenMS/contrib#122]
```
/usr/bin/oslevel =

  /usr/convex/getsysinfo =



  UNAME_MACHINE = aarch64

  UNAME_RELEASE = 5.15.0-1021-oracle

  UNAME_SYSTEM = Linux

  UNAME_VERSION = #27-Ubuntu SMP Fri Oct 14 20:04:20 UTC 2022

  configure: error: cannot guess build type you must specify one

  configure: error: /bin/bash './configure' failed for Data/Sample
Call Stack (most recent call first):
  CMakeLists.txt:590 (OPENMS_CONTRIB_BUILD_COINOR)
```

Solution:

`find . -name "config.guess" -exec cp /usr/share/automake-1.16/config.guess {} \;` or whatever, newer current automaker version you have installed.

**Comment by jpfeuffer** [comment]
Yes it is. It has been restructured into multiple standalone modules for years now and they use their own handwritten build system (lol). I would not invest time in this and instead integrate a different solver.
It works in conda but changing the contrib to build all the different submodules separately will take a lot of your time.

**Comment by cbielow** [comment]
https://github.com/ERGO-Code/HiGHS/tree/latest?

Benchmarks show it's around the same performance as CLP (with quite a bit of variance, depending on dataset)
